src/DankCord/DankCord.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, token: str, channel_id: id):
 
     self.gateway = Gateway(self.token)
     self.ws = self.gateway.ws
+    self.logger = self.gateway.logger
```

Comment:
better to instantize the logger in the client class and pass it to Gateway instead of the other way round

Reply:
It's in the gateway first since it's the first one to use it, before Client. Once the bot is ready, the logger is updated with the JSON data of the account, and the Client class uses it.